pkg/storage/statestore.go

```diff
@@ -29,6 +31,9 @@ type StateStorer interface {
 
 	// Iterate iterates over all keys with the given prefix and calls iterFunc.
 	Iterate(prefix string, iterFunc StateIterFunc) error
+
+	// Nuke the store so that only the bare essential entries are left.
+	Nuke(forgetStamps bool) error
```

Comment:
Lets move this into a different interface. We dont really need this for in-mem or other implementations.